crc/services/workflow_service.py

```diff
@@ -238,6 +239,26 @@ def test_spec(spec_id, validate_study_id=None, test_until=None, required_only=Fa
             WorkflowService.delete_test_data(workflow_model)
         return processor.bpmn_workflow.last_task.data
 
+    @staticmethod
+    def validate_form(task, task_api):
+        for field in task_api.form.fields:
+            if WorkflowService.is_required_field(field, task):
+                if not field.id in task.data or task.data[field.id] is None:
+                    return False
+        return True
+
```

Comment:
Just a question for me.
What's the difference/trade-off of having a static method and calling the class inside it versus a regular method and using self?